OpenXmlPowerTools/SpreadsheetWriter.cs

```diff
@@ -584,49 +622,72 @@ private static bool MatchAlignment(XDocument sXDoc, XElement xf, CellDfn cell)
             if ((int?)xf.Attribute(SSNoNamespace.applyAlignment) == 0 ||
                 (xf.Attribute(SSNoNamespace.applyAlignment) == null) &&
                 cell.HorizontalCellAlignment == null)
+            {
                 return true;
+            }
+
             if (xf.Attribute(SSNoNamespace.applyAlignment) == null &&
                 cell.HorizontalCellAlignment != null)
+            {
                 return false;
-            string alignment = (string)xf.Element(S.alignment).Attribute(SSNoNamespace.horizontal);
-            bool match = alignment == cell.HorizontalCellAlignment.ToString().ToLower();
+            }
+
+            var alignment = (string)xf.Element(S.alignment).Attribute(SSNoNamespace.horizontal);
+            var match = alignment == cell.HorizontalCellAlignment.ToString().ToLower();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/gh/Codeuctivity/Open-Xml-PowerTools/pullRequest?prid=5308324)